### PR TITLE
[GH-17954] migrate profile_button to styled component

### PR DESCRIPTION
**Pull request by lex111**
#### Summary

This will migrate webapp/src/components/profile/profile_button.scss to styled-components

#### Ticket Link

Fixes: https://github.com/mattermost/mattermost-server/issues/17954

#### Checklist

~~- [ ] Telemetry updated~~
~~- [ ] Gated by experimental feature flag~~
~~- [ ] Unit tests updated~~

### CODE REVIEW

File: webapp/src/components/profile/profile_button.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import styled from 'styled-components';
 
 import './profile_button.scss';
```
Comment (crspeller) [inline webapp/src/components/profile/profile_button.tsx:7]: Looks like you just forgot to remove the import. That's why the build is failing.

**Review by crspeller — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: webapp/src/components/profile/profile_button.tsx (hunk 1 above)
Comment (lex111) [reply webapp/src/components/profile/profile_button.tsx]: Oh, that's right, I forgot all about that.

**Review by crspeller — APPROVED** [review]

**Comment by emilyacook** [comment]
Thanks for participating in Hacktoberfest! You can claim your sticker set here: https://get.printfection.com/hacktober21/4144583267 @lex111

**Pull request merged**